Микросетки на флексах для всех страниц

Developer2220 opened the pull request:
---
:mortar_board: [Микросетки на флексах для всех страниц](https://up.htmlacademy.ru/htmlcss/36/user/2302197/tasks/12)

Developer2220 committed:
создает сетку компонентов на главной странице

Developer2220 committed:
создает сетку компонентов на внутренней странице

Developer2220 committed:
выравнивает код

Developer2220 committed:
добавляет гриды на внутреннюю страницу

Kartaviy-arteom commented on catalog.html:
```diff
@@ -11,7 +11,7 @@
       <nav class="navigation">
         <img class="logo" src="images/logo.svg" width="140" height="70" alt="Логотип города «Седона»">
-        <ul class="">
+        <ul class="navigation-list">
```
Проверить переполнение

Kartaviy-arteom commented on styles/styles.css:
```diff
@@ -94,7 +96,7 @@ body {
   width: 1200px;
   margin: 0 auto;
-
+  flex-grow: 1;
```
.hero нужен еще фоновый цвет

Kartaviy-arteom commented on styles/styles.css:
```diff
@@ -220,24 +223,31 @@ body {
   justify-content: center;
```
переполнение проверить

Kartaviy-arteom reviewed (commented):
Выравнивание содержимого в карточках подправить
Например .advantages-item .advantages-title {
    padding-top: 102px;
} - это не лучшее решение
Переполнение карточек, кнопок сортировки и тд

Developer2220 committed:
исправляет замечания по микросеткам

выравнивает содержимое в карточках , переполнение карточек, кнопок сортировки, добавляет фон hero

Developer2220 committed:
добавляет управление переполнением в сортировке

Developer2220 committed:
добавляет управление переполнением социальным сетям в футере

Developer2220 committed:
выравнивает текст в кнопках, исправляет размер шрифта

Developer2220 committed:
добавляет кнопкам cursor:pointer
